src/api/provider.ts

```diff
@@ -5,6 +5,7 @@ export interface APIResponse {
     json?: any
     text: string
     status: number
+    headers: Record<string, unknown>
```

Comment:
Why unknown here? We know that headers always are strings right? Setting it to unknown will force callers to explicitly cast each header value to a string before it can be used or typescript will shout at them

Reply:
I didn't see in the spec that it was explicitly a string, so I figured this was safer. Having typescript yell at people about making them strings however isn't a good side effect. Maybe I set it as a string, and then here: 

https://github.com/greymass/eosio-core/blob/f25892065d4ab89bce64a47b844e2f2f2a3d415b/src/api/provider.ts#L66

I force it to a string? e.g. 

```js
headers[key] = String(value)
```

Reply:
[Spec says string](https://fetch.spec.whatwg.org/#headers) 👍 Should be good without forcing it

Reply:
Ah so it does! Cool, change will be pushed in a moment.